code/modules/mob/mob.dm

```diff
@@ -42,6 +42,7 @@
 				hud_list[hud] = list()
 			else
 				var/image/I = image('icons/mob/hud.dmi', src, "")
+				I.appearance_flags = RESET_COLOR | RESET_ALPHA
```

Comment:
Won't this also make invisible people visible on the HUDs? Like with the genetics chameleon power?

Reply:
Yeah, this resetting alpha isn't ok.

It not inheriting color is fine.

Reply:
Fixed